instances/treasury-devdao.near/widget/pages/members/index.jsx

```diff
@@ -98,7 +102,7 @@ const [showDeleteModal, setShowDeleteModal] = useState(false);
 
 useEffect(() => {
   setLoading(true);
-  if (!loading) {
+  if (!loading && typeof getMembersAndPermissions === "function") {
```

Comment:
what happens if getMembersAndPermission changes (when loaded from the vm.require)? will the effect be triggered then?

Reply:
for the first render it will be undefined, and then it will be a function, it wouldn't change after that.
The reason to add it like this, earlier I defined it as an empty function, but then the loader kept on loading since the fn was empty and this effect runs only once, but now it will update when the fn changes